src/TLabel.cpp

```diff
@@ -67,9 +99,12 @@ void TLabel::setLeave(Host* pHost, const QString& func, const TEvent& args)
 
 void TLabel::mousePressEvent(QMouseEvent* event)
 {
-    if (event->button() == Qt::LeftButton) {
+    if (mMouseButtons.contains(event->button())) {
         if (mpHost) {
-            mpHost->getLuaInterpreter()->callEventHandler(mClick, mClickParams);
+            TEvent tmpClickParams = mClickParams;
```

Comment:
ℹ️ For many classes we have disabled the copy constructor but fortunately the `TEvent` class is not one of them - otherwise you would not be able to do this... 😉